core/connection/session_storage.go

```diff
@@ -41,9 +43,10 @@ func (repo *SessionStorage) Save(se Session) error {
 }
 
 // Update updates specified fields of existing session by id
-func (repo *SessionStorage) Update(sessionID session.ID, duration int, dataStats stats.SessionStats) error {
-	// update two fields by sessionID
-	se := Session{SessionID: sessionID, Duration: duration, DataStats: dataStats}
+func (repo *SessionStorage) Update(sessionID session.ID, TimeUpdated time.Time, dataStats stats.SessionStats,
+	status SessionStatus) error {
```

Comment:
How does this formatted, why it splitted to 2 lines?

Reply:
fixed